compose/compose-file.md

```diff
@@ -256,9 +256,11 @@ updates.
 - `monitor`: Duration after each task update to monitor for failure `(ns|us|ms|s|m|h)` (default 0s).
 - `max_failure_ratio`: Failure rate to tolerate during an update.
 
+```
```

Comment:
@johndmulhausen if you wish to use this syntax, you have to remove the whitespace in front